cantor-archive/src/main/java/com/salesforce/cantor/archive/file/ArchiverOnFile.java

```diff
@@ -0,0 +1,68 @@
+package com.salesforce.cantor.archive.file;
+
+import com.salesforce.cantor.misc.archivable.CantorArchiver;
+import com.salesforce.cantor.misc.archivable.EventsArchiver;
+import com.salesforce.cantor.misc.archivable.ObjectsArchiver;
+import com.salesforce.cantor.misc.archivable.SetsArchiver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+
+import static com.salesforce.cantor.common.CommonPreconditions.checkArgument;
+
+/**
+ * An implementation of the archiver which stores data on disk.
+ *
+ * {@code baseDirectory} and {@code chunkMillis} can specified or will default to
+ * {@link ArchiverOnFile#defaultArchivePathBase} and {@link ArchiverOnFile#defaultChunkMillis} respectively
+ */
+public class ArchiverOnFile implements CantorArchiver {
+    private static final Logger logger = LoggerFactory.getLogger(ArchiverOnFile.class);
+
+    protected static final String defaultArchivePathBase = "/tmp/cantor-archive";
```

Comment:
@AlexanderKouthoofd change this to just `cantor-archive` instead of `/tmp/...`

Reply:
Makes sense. Changed.